instrumentation/runtime-metrics/runtime-metrics-java17/library/src/test/java/io/opentelemetry/instrumentation/runtimemetrics/java17/RuntimeMetricsTest.java

```diff
@@ -92,11 +92,13 @@ void close() {
       jfrTelemetry.close();
       logs.assertDoesNotContain("RuntimeMetrics is already closed");
       assertThat(recordingStreamClosed.get()).isTrue();
-      assertThat(reader.collectAllMetrics())
-          .allSatisfy(
-              metric -> {
-                assertThat(metric.getName()).isEqualTo("process.runtime.jvm.gc.duration");
```

Comment:
I think the original assertion might've reflected the #8475 issue -- the GC metrics aren't reliably closed in the current shape of the library